testing/integration/network_traffic_monitoring_test.go

```diff
@@ -0,0 +1,121 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License 2.0;
+// you may not use this file except in compliance with the Elastic License 2.0.
+
+//go:build integration
+
+package integration
+
+import (
+	"context"
+	"encoding/json"
+	"testing"
+	"time"
+
+	"github.com/gofrs/uuid/v5"
+	"github.com/stretchr/testify/require"
+	"github.com/stretchr/testify/suite"
+
+	"github.com/elastic/elastic-agent-libs/kibana"
+	"github.com/elastic/elastic-agent-libs/testing/estools"
+	atesting "github.com/elastic/elastic-agent/pkg/testing"
+	"github.com/elastic/elastic-agent/pkg/testing/define"
+	"github.com/elastic/elastic-agent/pkg/testing/tools"
+)
+
+type NetworkTrafficRunner struct {
+	suite.Suite
+	info         *define.Info
+	agentFixture *atesting.Fixture
+
+	ESHost string
+}
+
+func TestNetworkTraffic(t *testing.T) {
+	info := define.Require(t, define.Requirements{
+		Group: Fleet,
+		Stack: &define.Stack{},
+		Local: false, // requires Agent installation
+		Sudo:  true,  // requires Agent installation
+		OS: []define.OS{
+			{Type: define.Linux},
+			{Type: define.Windows},
```

Comment:
if these two tests run concurrently at the same time the fact that they install the same package is it gonna affect any of them? I am not sure if the same as with endpoint test will happen here or whether kibana should handle that just wondering

Reply:
should use different policy right?

Reply:
Installing a package should work concurrently. If installing a package doesn't work concurrently and it errors back to the caller, then the caller should retry. Have we added retries?

Reply:
@blakerouse retries in a test itself?

Reply:
I wouldn't expect it to be something the test itself should do. Probably something the kibana client that the test uses should do. If it gets something like a conflict error (or what ever error that is reported by kibana, hopefully not a 500 error) then it should retry those calls automatically.

Reply:
ok to me then it means that this PR doesn't have to include anything else, we just need to make sure that kibana client does retry. @blakerouse should we resolve this one?

Reply:
Yeah I didn't mean for this to block this PR. Just a comment that packing installation should add retries in the case of failures working with Kibana API.